File: Python Programs/Armstrong.py
n = int(input("Enter a number: "))
c=n
s=0
k=0
while c!=0:
    k+=1
    c//=10
c=n
while c!=0:
    d=c%10
    s+=d**k
    c//=10
print("Armstrong number" if s==n else "Not an armstrong number")

File: Java Projects/9th/Armstrong.java
import java.util.Scanner;

public class Armstrong {
    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            System.out.println("Enter a number");
            int n = scanner.nextInt();
            int c=n, k=0;
            while (c!=0) {
                k++;
                c/=10;
            }   c=n;
            int s=0;
            while (c!=0) {
                int d=c%10;
                s=s+(int)Math.pow(d,k);
                c/=10;
            }   System.out.println(n==s?"Armstrong number":"Not a armstring number");
        }
    }
}
